cms/server/admin/static/admin_modals.js

```
@@ -0,0 +1,249 @@
+/* Contest Management System
+ * Copyright © 2024 IOI-ISR
+ *
+ * Centralized modal management using MicroModal.
+ * SweetAlert2 is used as a drop-in replacement for native confirm() dialogs.
+ * Provides global initialization, URL-driven auto-open, and generic
+ * confirm/delete helpers so individual templates don't duplicate logic.
+ */
+
+"use strict";
+
+var AdminModals = AdminModals || {};
```

Biome (2.3.13)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)